controllers/userController.js

```diff
@@ -57,4 +57,64 @@ const emailCheck = async (req, res) => {
   }
 };
 
-module.exports = { signUp, signIn, emailCheck };
+const myPageShow = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.mypage(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const myAddress = async (req, res) => {
+  const email = req.user.email;
+  try {
+    const result = await userService.myAddressShow(email);
+    return res.status(200).json({ data: result });
+  } catch (err) {
+    console.log(err);
+    return res.status(err.statusCode || 500).json({ message: err.message });
+  }
+};
+
+const addressPatch = async (req, res) => {
```

Comment:
```suggestion
const updateUserAddress = async (req, res) => {
```
동작과 자원의 조합으로 작성되면 일관적으로, 직관적으로 작성될 수 있습니다 :)

Reply:
네 확인하고 수정했습니다 !